packages/flutter_adaptive_scaffold/lib/src/adaptive_scaffold.dart

```diff
@@ -305,14 +305,15 @@ class AdaptiveScaffold extends StatefulWidget {
   /// a list of [NavigationDestination]s.
   static Builder standardBottomNavigationBar({
     required List<NavigationDestination> destinations,
-    int currentIndex = 0,
+    int? currentIndex,
     double iconSize = 24,
     ValueChanged<int>? onDestinationSelected,
   }) {
+    currentIndex ??= 0;
     return Builder(
       builder: (_) {
         return BottomNavigationBar(
-          currentIndex: currentIndex,
+          currentIndex: currentIndex!,
```

Comment:
Don't bother to assign it above, just do this:

```suggestion
          currentIndex: currentIndex ?? 0,
```